docs/getting_started/troubleshooting.rst

```diff
@@ -209,13 +210,87 @@ This can happen when you use OmniSci engine along with ``pyarrow.gandiva``:
   cfg.IsExperimental.put(True)
   import modin.pandas as pd
   import pyarrow.gandiva as gandiva  # Error
-  CommandLine Error: Option 'enable-vfe' registered more than once!
-  LLVM ERROR: inconsistency in registered CommandLine options
-  Aborted (core dumped)
+  # CommandLine Error: Option 'enable-vfe' registered more than once!
+  # LLVM ERROR: inconsistency in registered CommandLine options
```

Comment:
Is this PR supposed to touch this file?

Reply:
No. I think this is a GitHub bug in the PR page. If I pull from GitHub and diff against master, I don't see this diff. These lines are in master [here](https://github.com/modin-project/modin/blob/master/docs/getting_started/troubleshooting.rst).

Reply:
This has happened to me before.

Reply:
it probably happened, because your branch introduces some commits from the master branch that are not part of any merge-commit:

<details><summary>screenshot</summary>

![image](https://user-images.githubusercontent.com/62142979/152943951-b770d068-a756-4f5b-9cc9-d58bf049d19c.png)

</details>

I would suggest to try to rebase on master

Reply:
Yes, that works. The spurious diffs are gone.